autometa/__main__.py

```diff
@@ -184,14 +186,16 @@ def entrypoint():
         sys.exit(1)
     except Exception as err:
         issue_request = """
+        An error was encountered!
 
         Please help us fix your problem!
 
         You may file an issue with us at https://github.com/KwanLab/Autometa/issues/new/choose
         """
-        err.issue_request = issue_request
+        # err.issue_request = issue_request
         logger.exception(err)
-        logger.info(err.issue_request)
+        logger.info(issue_request)
+        sys.exit(1)
```

Comment:
These changes are invalid. You are no longer raising the original error that was encountered. This is going to make troubleshooting/debugging quite difficult

Reply:
My formatter was showing the following error for `logger.info(err.issue_request)`, "Instance of 'Exception' has no 'issue_request'".
I could be wrong about this but why can't we just use: `logger.info(issue_request)` as `logger.exception(err)` has already logged the complete traceback.
Also, this is just catching the error right? I was thinking of using `sys.exit(1)` to exit as in `KeyboardInterrupt` above.
Let me know what you think about the changes.

Reply:
Hi Sidd, I must have missed the `logger.exception(err)`. My mistake. You are right, this should still emit the traceback and the issue request to the user.

Reply:
Alright. I think we should also include `sys.exit(1)` to exit the pipeline in case the error is encountered. Or am I missing something?